build.md

```diff
@@ -227,6 +227,28 @@ Cache target is defined using the same `type=TYPE[,KEY=VALUE]` syntax defined by
 
 Unsupported cache target MUST be ignored and not prevent user from building image.
 
+### additional_contexts
+
+`additional_contexts` defines a list of named contexts the image builder SHOULD use during image build.
```

Comment:
Should we specify the behavior if the builder doesn't support additional contexts? Warning or error? 🤔

Reply:
Do we do this for other features of this type?

Reply:
[The spec's default rule](https://github.com/compose-spec/compose-spec/blob/4ccbdd71694d97c3b011daba6f1d713e745c7807/spec.md?plain=1#L23) lets the implementation decide what to do with optional attributes.
But we can add something like that
`The Compose implementation SHOULD warn that the builder does not support additional contexts and MAY list the unused contexts.`

Reply:
Sounds good, thanks for the suggestion! I updated the description to include it.